DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: phiproject
-Title: PHI R Project Template
+Title: PHI R Templates
 Authors@R: c(
     person("Anna", "Price", email = "anna.price5@nhs.net", role = c("aut", "cre")),
     person("Alan", "Yeung", email = "alan.yeung@nhs.net", role = "aut", comment = c(ORCID = "0000-0001-5226-3695"))
```

Comment:
Surely you can take on the job of maintainer now @alan-y? 👀

Reply:
haha I suppose I could update this but I wanted to make sure Anna is credited with her contribution first